app/custom-oauth/server/custom_oauth_server.js

```diff
@@ -303,6 +306,9 @@ export class CustomOAuth {
 				if (!user) {
 					return;
 				}
+				if (this.mergeRoles) {
+					mapRolesFromSSO(user, serviceData, this.rolesClaim || 'roles');
```

Comment:
```suggestion
					mapRolesFromSSO(user, serviceData, this.rolesClaim);
```

Un-needed since setting at this.rolesClaim definition